rtlib/include/rt/Image.hpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by Anthony Tamasi on 2020-02-04.
+//
+
+#pragma once
+
+#include <boost/filesystem.hpp>
+#include <opencv2/core.hpp>
+
+#include "rt/Data.hpp"
+
+
+namespace rt {
+    /**
+     * @class Image
+     * @brief Represent a simple 2D image for the registration pipeline
+     * @ingroup data
+     */
+    class Image : public Data
+    {
+    public:
+        /** Constructor */
+        explicit Image(const boost::filesystem::path& path);
+
+        /** @brief Return a single image */
+        cv::Mat getImage(int idx = 0) { return img_; }
```

Comment:
Move all implementation details to the CPP

Reply:
Even the one-liner stuff? I thought you had said to just leave getters/setters in the hpp.